.github/actions/build-dotnet/action.yml

```diff
@@ -137,11 +137,19 @@ runs:
         key: ${{ steps.detect-cache-configuration.outputs.key }}
         restore-keys: ${{ steps.detect-cache-configuration.outputs.restore-keys }}
 
+    - name: Create temporary global.json with desired .NET SDK version
+      shell: pwsh
+      working-directory: ${{ inputs.directory }}
+      run: |
+        "{ ""sdk"": { ""version"": ""$Env:DOTNET_SDK_TO_USE"" } }" > ./global.json
```

Comment:
Only create a file if it doesn't exist (this is going into the root of the repo clone, right?).

Reply:
That's correct.